vmaas/reposcan/exporter.py

```diff
@@ -748,6 +749,59 @@ def _dump_oval(self, dump):
                 dump.execute("insert into oval_module_test_detail values (?, ?)",
                              (oval_module_test_id, module_stream))
 
+    def _dump_csaf(self, dump):
+        """Select CSAF data"""
+        # Dump data from csaf_product table
+        dump.execute("""CREATE TABLE IF NOT EXISTS csaf_product (
+                        id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        cpe_id INTEGER NOT NULL,
+                        package_name_id INTEGER,
+                        package_id INTEGER,
+                        module_stream_id INTEGER
+                    )""")
+        with self._named_cursor() as cursor:
+            cursor.execute("""SELECT *
+                            FROM csaf_product""")
+            for csaf_product_id, cpe_id, package_name_id, package_id, module_stream in cursor:
+                dump.execute("INSERT INTO csaf_product VALUES (?, ?, ?, ?, ?)",
+                             (csaf_product_id, cpe_id, package_name_id, package_id, module_stream))
+
+        # Dump data from csaf_cve_product table
+        dump.execute("""CREATE TABLE IF NOT EXISTS csaf_cve_product (
+                        id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        cve_id INTEGER NOT NULL,
+                        csaf_product_id INTEGER NOT NULL,
+                        csaf_product_status_id INTEGER NOT NULL,
+                        csaf_file_id INTEGER NOT NULL)
+                    """)
+        with self._named_cursor() as cursor:
+            cursor.execute("""SELECT *
+                            FROM csaf_cve_product""")
+            for csaf_cve_id, cve_id, csaf_product_id, csaf_product_status_id, csaf_file_id in cursor:
+                dump.execute("INSERT INTO csaf_cve_product VALUES (?, ?, ?, ?, ?)",
+                             (csaf_cve_id, cve_id, csaf_product_id, csaf_product_status_id, csaf_file_id))
+
+        # Dump data from csaf_file table
+        dump.execute("""CREATE TABLE IF NOT EXISTS csaf_file (
+                        id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        name TEXT UNIQUE NOT NULL,
+                        updated TIMESTAMP)""")
```

Comment:
and if csaf_file_id is not needed, then we can exclude the whole csaf_file table from the dump